server/src/mixer.rs

```diff
@@ -341,7 +341,7 @@ impl Mixer {
     fn update_mixing_state(
         agg: &gst_base::Aggregator,
         id: &str,
-        pts: gst::ClockTime,
+        pts: Option<gst::ClockTime>,
```

Comment:
Unclear if this always expects a valid PTS or if it can allow `None` too. This might need changing.